### PR TITLE
Docs: add FAQ about vue-eslint-parser

### PR DESCRIPTION
**Pull request by mysticatea**
This PR adds FAQ about "Use the latest vue-eslint-parser" errors.
Also, the error message points that.

Please check this description is enough or not.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,30 @@ The `--fix` option on the command line automatically fixes problems reported by
 
 <!--RULES_TABLE_END-->
 
+## :couple: FAQ
+
+### What is "Use the latest vue-eslint-parser" errors?
```
Comment (michalsnik) [inline README.md]: I think it should be:
```
What is the `Use the latest vue-eslint-parser` error?
```

File: README.md [hunk 2]
```diff
@@ -113,6 +113,30 @@ The `--fix` option on the command line automatically fixes problems reported by
...
+
+The most rules of `eslint-plugin-vue` requires `vue-eslint-parser` to check `<template>` ASTs.
+
+Please add one of the following settings into your configuration:
```
Comment (michalsnik) [inline README.md]: What do you think about: `Make sure you have one of the following settings in your **.eslintrc.js**:` ?

File: README.md [hunk 3]
```diff
@@ -113,6 +113,30 @@ The `--fix` option on the command line automatically fixes problems reported by
...
+
+- `"extends": ["plugin:vue/recommended"]`
+- `"extends": ["plugin:vue/base"]`
+- `"parser": "vue-eslint-parser"` (and do `npm install -D vue-eslint-parser`)
```
Comment (michalsnik) [inline README.md]: I think we can remove the third option and force using base or recommended configuration. The less options the better, people will not get confused :)

File: README.md [hunk 4]
```diff
@@ -113,6 +113,30 @@ The `--fix` option on the command line automatically fixes problems reported by
...
+
+If you have `"parser": "babel-eslint"` setting or something like in your configuration, please move it into `parserOptions`.
```
Comment (michalsnik) [inline README.md]: If you already use other parser (e.g. `"parser": "babel-eslint"`), please move it into `parserOptions`, so it doesn't collide with the `vue-eslint-parser` used by this plugin's configuration:

File: README.md [hunk 5]
```diff
@@ -113,6 +113,30 @@ The `--fix` option on the command line automatically fixes problems reported by
...
+The `vue-eslint-parser` uses the parser which is set by `parserOptions.parser` to parse scripts.
```
Comment (michalsnik) [inline README.md]: Please move this line after the diff.

**Comment by michalsnik** [comment]
Good call with the FAQ part @mysticatea  👍 I added few suggestions above

**Comment by mysticatea** [comment]
Thank you!
I updated this PR as the correcting.

**Pull request merged**